ui/app/AppLayouts/Wallet/controls/ManageTokensGroupDelegate.qml

```diff
@@ -13,18 +13,18 @@ DropArea {
     id: root
     objectName: "manageTokensGroupDelegate-%1".arg(index)
 
-    // expected roles: communityId, communityName, communityImage
+    // expected roles: communityId, communityName, communityImage, collectionUid, collectionName, imageUrl // FIXME unify group image
 
     property int visualIndex: index
     property var controller
-    property bool communityGroupsExpanded
     property var dragParent
     property alias dragEnabled: groupedCommunityTokenDelegate.dragEnabled
-    property bool isCollectible
+    property bool isCollectible: isCollection
```

Comment:
Not clear to me this initialization. Can you explain me what's that?

Reply:
Wondering if the component should be more generic and without adding the concept of "collection" since at the end it's just a group delegate definition. WDYT? 

Same thought with the "communities" related words.. here I see it more like a generic group delegate, doesn't matter if it's representing a community group or a collection group.

Reply:
Agree on making the delegates more generic; I'll do that is a followup branch just not to inflate this PR